features/kafka/producer/pom.xml

```diff
@@ -57,6 +57,43 @@
       <artifactId>org.opennms.core.lib</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.opennms</groupId>
+      <artifactId>opennms-dao-api</artifactId>
```

Comment:
Do we need this? I only see new imports for `org.springframework.expression.*` in the main module.

Reply:
Looks like blueprint needs it.